apps/fluent-tester/README.md

```diff
@@ -1,51 +1,60 @@
-`RNTester` is the test app that we use to test our FluentUI components during development.
+`FluentUI Tester` is the test app that we use to test our FluentUI components during development.
 
-## Launch `RNTester` app
+## Launch `FluentUI Tester` app
 
-1. After you've cloned the repository for the first time, follow the instructions [here](../../README.md) to install npm packages and build the entire repository.
+1. Make sure you have followed the Getting Started instructions [here](../../README.md) to install packages and build the entire repository.
 
-2. Then go into `experiments\tester` folder:
+2. Then go into `apps\fluent-tester` folder:
 
 ```
-    cd experiments\tester
+    cd apps\fluent-tester
 ```
 
-3. Build the RNTester bundle:
+3. Build the FluentUI Tester bundle:
 
 ```
     yarn bundle
```

Comment:
This actually builds both win32 and UWP.  This could be updated to be yarn bundle --platform win32 to just bundle that one.